### PR TITLE
Render summaries as markdown

### PR DESCRIPTION
**Pull request by rossjones**
.. because they're markdown.

Currently all of the summaries are markdown, but ignore that and end up
squashed into a single paragraph/wall-of-text.

This PR renders the markdown to the page which means it will autolink,
and keep the appropriate spacing.

We can turn off certain features, so we can stop people using some
markdown features.

https://trello.com/c/HoH94aZv/363-markdown-in-dataset-description

### CODE REVIEW

**Comment by rossjones** [comment]
@nicholsj This PR is just one so that we can see what happens when we render the summaries correctly - maybe we can discuss it some time next week?

File: app/helpers/datasets_helper.rb [hunk 1]
```diff
@@ -29,6 +29,11 @@ def displayed_date(dataset)
     end
   end
 
+  def to_markdown(content)
+    markdown = Redcarpet::Markdown.new(Redcarpet::Render::HTML, autolink: true, tables: false)
```
Comment (andrewgarner) [inline app/helpers/datasets_helper.rb]: @rossjones did you look at using https://kramdown.gettalong.org/rdoc/ – it's the library used by https://github.com/alphagov/govspeak

Comment (rossjones) [reply app/helpers/datasets_helper.rb]: Ah no, I just used redcarpet through familiarity (I've used it before).
We probably should switch for consistency

Comment (andrewgarner) [reply app/helpers/datasets_helper.rb]: Another option maybe would be `Govspeak::Document.new(content).to_sanitized_html`

Comment (hannako) [reply app/helpers/datasets_helper.rb]: @rossjones yes no probs

@andrewgarner ta

**Pull request merged**